### PR TITLE
Add WS subscribe for TX added to chain

### PR DESCRIPTION
**Pull request by hanssv**
+ tests

[Finishes #154800190]

### CODE REVIEW

**Comment by hanssv** [comment]
Dialyzer is not happy...

**Comment by hanssv** [comment]
Made dialyzer happy by writing the code differently... opaque types are not always great...

**Comment by hanssv** [comment]
Needs a rebase on #686

**Comment by hanssv** [comment]
Rebased, dialyzer is happy - ready for review...

File: apps/aehttp/src/ws_handler.erl [hunk 1]
```diff
@@ -24,13 +24,12 @@ init({tcp, http}, _Req, _Opts) ->
     {upgrade, protocol, cowboy_websocket}.
 
 websocket_init(_TransportName, Req, _Opts) ->
-    {_QsVals, Req1} = cowboy_req:qs_vals(Req),
```
Comment (velzevur) [inline apps/aehttp/src/ws_handler.erl]: Thanks

File: apps/aecore/src/aec_tx.erl [hunk 2]
```diff
@@ -111,7 +111,6 @@ calculate_total_fee(SignedTxs) ->
               TotalFee + Fee
       end, 0, SignedTxs).
 
--spec hash_tx(tx()) -> tx_hash().
```
Comment (velzevur) [inline apps/aecore/src/aec_tx.erl]: Why did you remove the spec?

**Comment by hanssv** [comment]
Pushed an update to swagger.yaml - did change the return format of some HTTP APIs
